qds_sdk/nezha.py

```diff
@@ -6,169 +6,251 @@
+        parser_datasource.add_argument(
+            "--type", dest="type", help="Type of data source")
+        parser_datasource.add_argument(
+            "--name", dest="name", help="Name of the data source")
+        parser_datasource.add_argument(
+            "--user", dest="user", help="Username for data source")
+        parser_datasource.add_argument(
+            "--password", dest="password", help="Password for data source")
+        parser_datasource.add_argument(
+            "--auth_token", dest="auth_token",
+            help="Authentication token for qds")
+        parser_datasource.add_argument(
+            "--dbtap_id", dest="dbtap_id", help="Id of the data-source in qds")
+
+        parser_cube = update_subparsers.add_parser(
+            "cubes", help="update cubes")
+        parser_cube.add_argument(
+            "id", help="Numeric id of the Cube")
+        parser_cube.add_argument(
+            "--cost", dest="cost", help="Cost for cube")
+        parser_cube.add_argument(
+            "--description", dest="description",
+            help="Description of the cube")
+        parser_cube.add_argument(
+            "--name", dest="name", help="Name of the Cube")
+        parser_cube.add_argument(
+            "--schema_name", dest="schema_name", help="Schema name")
+        parser_cube.add_argument(
+            "--table_name", dest="table_name", help="Table name")
+        parser_cube.add_argument(
+            "--grouping_column", dest="grouping_column",
+            help="Grouping Column")
+        parser_cube.add_argument(
+            "--destination_id", dest="destination_id",
+            help="Data Source id acting as destiantion")
+        parser_cube.add_argument("--query", dest="query", help="Query")
+
+        parser_partition = update_subparsers.add_parser(
+            "partitions", help="update partitions")
+        parser_partition.add_argument(
+            "id", help="Numeric id of the Partition")
+        parser_partition.add_argument(
+            "--cost", dest="cost", help="Cost for partition")
+        parser_partition.add_argument(
+            "--description", dest="description",
+            help="Description of the partition")
+        parser_partition.add_argument(
+            "--name", dest="name", help="Name of the Partition")
+        parser_partition.add_argument(
+            "--schema_name", dest="schema_name", help="Schema name")
+        parser_partition.add_argument(
+            "--table_name", dest="table_name", help="Table name")
+        parser_partition.add_argument(
+            "--destination_id", dest="destination_id", help="Data Source id")
+        parser_partition.add_argument("--query", dest="query", help="Query")
+
+        parser_ds = update_subparsers.add_parser(
+            "default_datasource", help="update default-datasource")
+        parser_ds.add_argument(
+            "default_datasource_id", help="Numeric id of the DataSource")
```

Comment:
What does `update default_datasource ID` do? Does it change the default datasource to be a different ID? If that's the case, we should explain it in a better way.